doc/contributor/project-layout.md

```diff
@@ -0,0 +1,33 @@
+# Project Layout
+
+Each folder in the TruffleRuby project is described below:
+
+## `/bench/`
```

Comment:
`##` renders huge on GitHub.
Maybe a lower-level title or just a list would be more readable?

Reply:
Fixed to `###`.